Makefile

```diff
@@ -9,6 +9,12 @@
 #     For correctness and performance tests this include halide build time and run time. For
 #     the tests in test/generator/ this times only the halide build time.
 
+# Halide project version
+HALIDE_VERSION_MAJOR ?= 15
```

Comment:
Per earlier discussions elsewhere, you shouldn't be under any obligation to add Vulkan support to Make (CMake-only is fine)

Reply:
Ah, okay!  It wasn't clear if we were maintaining both ...